libs/SalesforceSDK/src/com/salesforce/androidsdk/security/SalesforceKeyGenerator.java

```diff
@@ -151,57 +137,39 @@ public static String getSHA256Hash(String privateKey) {
     }
 
     /**
-     * Generates a keypair and returns the public key.
+     * Generates an RSA keypair and returns the public key.
      *
      * @param name Alias of the entry in which the generated key will appear in Android KeyStore.
+     * @param length Key length.
      * @return RSA public key.
+     * @deprecated Will be removed in Mobile SDK 8.0. Use {@link KeyStoreWrapper} instead.
      */
     public static synchronized PublicKey getRSAPublicKey(String name, int length) {
```

Comment:
I marked these `public` methods `deprecated` and moved the implementation inside `KeyStoreWrapper`.